learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md

```diff
@@ -0,0 +1,128 @@
+# Integrating JUnit tests for WaveMaker application
+
+
+### OVERVIEW
+
+JUnit is a unit testing framework for the Java programming language. JUnit has been important in the development of test-driven development. These tests are required whether imported services are working as expected or not, before proceeding to run/deployment of the application. This document demonstrates how to integrate JUnit tests in an application developed by WaveMaker RAD.
+
+### GOALS
+
+1. Integrate JUnit tests in the WaveMaker application
+2. Writing sample test cases for services.
+
+### PREREQUISITES 
+
+1. IDE (IntelliJ IDEA, Eclipse)
+2. WaveMaker Application
+3. Git
+4. Maven
+
+### CONFIGURATION
+
+1. Export the WaveMaker application as a zip file.
```

Comment:
link to the associated document in learn site.

Reply:
Added the document to export WM project as a Zip